0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
> That would be entirely fine with me.

Cool!

> I'd just like to get this sorted out before we further propagate it.

I'd prefer to have this PR merged sooner rather than later, as until its merged, I'll have to add every new contributor to the list of people requried to sign off (I already had to add killercup as his RFC got merged after I made the initial list). And some people can't contribute as long as this PR is unmerged because they insisted on only agreeing to licensing their past (not past and future) contributions under the license...